addons/stock/models/stock_orderpoint.py

```diff
@@ -146,18 +257,178 @@ def _get_product_context(self):
             'to_date': datetime.combine(self.lead_days_date, time.max)
         }
 
-    def _prepare_procurement_values(self, product_qty, date=False, group=False):
+    def _get_orderpoint_action(self):
```

Comment:
adding a docstring explaining this is here than we generated the auto RR would be useful